main.py

```diff
@@ -1,7 +1,54 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        {"first_name": "George", "last_name": "Klooney"},
+        {"first_name": "Kianu", "last_name": "Reaves"},
```

Comment:
There is a typo in the actor's first name 'Kianu'. It should be 'Keanu' to match the intended update operation later in the script.